File: FSharp/FizzBuzz/fizzbuzz.fs
let FizzBuzz (i : int) =
    if   i % 15 = 0 then "FizzBuzz"
    elif i %  3 = 0 then "Fizz"
    elif i %  5 = 0 then "Buzz"
    else i.ToString()

let run (n : int) =
    for i = 1 to n do
        printfn "%s" (FizzBuzz i)

run 15

File: JavaScript/FizzBuzz/fizzbuzz.js
const fizzbuzz = (i) => {
    if(i % 15 === 0) {
        return "FizzBuzz";
    } else if (i % 3 === 0) {
        return "Fizz";
    } else if (i % 5 === 0) {
        return "Buzz";
    }
    return i;
};

const run = (n) => {
    for(let i = 1; i <= n; i++) {
        console.log(fizzbuzz(i));
    }
};

run(15);
